core/src/main/java/cucumber/api/event/ConcurrentEventListener.java

```diff
@@ -0,0 +1,18 @@
+package cucumber.api.event;
+
+/**
+ * This is the interface you should implement if your plugin listens to cucumber execution events
+ *
+ * TODO: Doc the concurrent part
+ *
+ */
+public interface ConcurrentEventListener {
```

Comment:
Maybe rename this to `ConcurrentEventAware` to match the `ColorAware` and `StrictAware` pattern?

Reply:
Depends how long `EventListener` is going to be around.  But I can see arguments both for keeping it inline with the other "Aware" interfaces, but also to name it similarly to the existing listener

Reply:
Not long at all. I want to deprecate it in the same release as parallel cukes. People will have to update their formatters, might as well do it properly.